backend/src/server/createServer.ts

```diff
@@ -21,9 +22,10 @@ export function createServer() {
   // Backend endpoints
   app.use('/authentication', authenticationRouter);
   app.use('/comment', mustBeAuthenticated, commentRouter);
+  app.use('/games', gameRoutes);
   app.use('/gamers', mustBeAuthenticated, gamerRoutes);
   app.use('/user', mustBeAuthenticated, userInfoRouter);
   app.use('/posts', mustBeAuthenticated, postRoutes);
-  app.use('/postTags', mustBeAuthenticated, postTagRoutes);
+  app.use('/postTags', postTagRoutes);
```

Comment:
:( why ?

Reply:
I had to remove it to add tags via Postman

Reply:
I don't think it is that useful either, there is nothing to do on post tag routes